monkey/infection_monkey/post_breach/signed_script_proxy/windows/signed_script_proxy.py

```diff
@@ -0,0 +1,16 @@
+import subprocess
+
+ORIGINAL_COMSPEC = r'C:\Windows\System32\cmd.exe'
```

Comment:
1. Use Pathlib
2. What if Windows isn't installed on c:? Maybe better to use %windir%

Reply:
Even better would be to leave this undefined and if we can't know what's the original comspec for some reason, don't touch it at all.